ports/espressif/boards/es3ink/mpconfigboard.mk

```diff
@@ -0,0 +1,10 @@
+USB_VID = 0x239A
+USB_PID = 0x2031
```

Comment:
You'll need your own USB PID. 0x239A is adafruit's VID. espressif gives them out: https://github.com/espressif/usb-pids

Reply:
Sorry, I forget to change it after I copied files to make definition for my board. It should be OK now. Thanks